mqtt/mqtt-edgehub/src/topic/translation.rs

```diff
@@ -176,6 +179,13 @@ macro_rules! translate_c2d {
 
 translate_d2c! {
     // Message Translation
+    module_send_message {
+        to_new_topic {
+            format!("devices/{}/modules/{}/messages/events(?P<path>.*)", DEVICE_ID, MODULE_ID),
```

Comment:
Instead of inlining the format, it seems like it will be cleaner to define the map in one central place.
Doesn't have to be done now though..

Reply:
Yeah, I don't like implementation too much. Still could not come up with anything better.